File: app/src/test/java/net/laurakogler/medminder/ExampleRobolectricTest.java
package net.laurakogler.medminder;

import android.widget.TextView;

import org.junit.Test;
import org.junit.runner.RunWith;
import org.robolectric.Robolectric;
import org.robolectric.RobolectricGradleTestRunner;
import org.robolectric.annotation.Config;

import static org.assertj.core.api.Assertions.assertThat;

@RunWith(RobolectricGradleTestRunner.class)
@Config(constants = BuildConfig.class)
public class ExampleRobolectricTest {

    @Test
    public void itShouldDisplayHelloWorld() {
        MainActivity activity = Robolectric.setupActivity(MainActivity.class);
        TextView textView = (TextView) activity.findViewById(R.id.main_text);

        assertThat(textView.getText()).isEqualTo("Hello World!");
    }
}

File: app/src/test/java/net/laurakogler/medminder/ExampleRobolectricTest.kt
package net.laurakogler.medminder

import android.widget.TextView

import org.junit.Test
import org.junit.runner.RunWith
import org.robolectric.Robolectric
import org.robolectric.RobolectricGradleTestRunner
import org.robolectric.annotation.Config

import org.assertj.core.api.Assertions.assertThat

@RunWith(RobolectricGradleTestRunner::class)
@Config(constants = BuildConfig::class)
class ExampleRobolectricTest {

    @Test
    fun itShouldDisplayHelloWorld() {
        val activity = Robolectric.setupActivity(MainActivity::class.java)
        val textView = activity.findViewById(R.id.main_text) as TextView

        assertThat(textView.text).isEqualTo("Hello World!")
    }
}
